pennylane/allocation.py

```diff
@@ -150,21 +150,17 @@ def deallocate(wires: DynamicWire | Wires | Sequence[DynamicWire]) -> Deallocate
 
     .. code-block:: python
 
-        @qml.qnode(qml.device('default.qubit'))
         def c():
             qml.H(0)
 
-            wires = qml.allocation.allocate(1, require_zeros=True)
+            wires = qml.allocate(1, require_zeros=True)
```

Comment:
These functions are not available top-level. Locally, I need to access them as `qml.allocation.allocate()` and `qml.allocation.deallocate()`